### ISSUE TITLE
Trying to get property 'retries' of non-object

### ISSUE THREAD

**Issue by DjThossi** [oneflow/oneflow-sdk-php#40]
We just got the new code from version 1.2.0 via composer update. 

Our first tries failed with the following error
> Trying to get property 'retries' of non-object
This blocked us from using this library as we are currently not able to send new orders to siteflow.

We identified the error in `OneflowSDK.php` in line 59 and following 
`$this->retries = $options->retries ?: 3;`
You must not try to directly access objects' attributes or methods if they can be null!

A simple fix for this would be
`$this->retries = isset($options->retries) ? $options->retries : 3;`

Keep in mind that you have the same issue in the next lines again for other attributes of the object.

Please let me know, as soon as you're done with the fix.

**Comment by juanjoseruiz** [comment]
Hi @DjThossi sorry for the late reply. I have not received a notification for this issue.

Which PHP version are you using? I tried in PHP 7.4 and the code works fine for me. 

If you can confirm your suggestion is working fine for your version, I will change it

**Comment by DjThossi** [comment]
Hey @juanjoseruiz 

Yes, we're using a pretty strict PHP version 7.4 with all errors, warnings, notices and deprecations active and we take care of them before we push our code to production. 

Here is an example to show you the problem
```
<?php
declare(strict_types=1);

error_reporting(E_ALL);
ini_set('display_errors', '1');

$options = null;

echo $options->retries ?: 3;
//echo isset($options->retries) ? $options->retries: 3;
```

If you run this code it should output this
```
PHP Notice:  Trying to get property 'retries' of non-object in /Users/sthoss/projects/bpp/oneflow-api/bug-example.php on line 9

Notice: Trying to get property 'retries' of non-object in /Users/sthoss/projects/bpp/oneflow-api/bug-example.php on line 9
3
```

If you comment out line 9 and comment in line 10 the result will be `3` without the Notice.

Do you see it now? Please let me know if I can be of any further help.

**Comment by juanjoseruiz** [comment]
Thanks @DjThossi I've just created a PR to fix the issue. Will let you know once merged and publish a new release.

**Comment by juanjoseruiz** [comment]
@DjThossi I just published a 1.3.0 release that should fix your problem. Please, feel free to reopen if you are still experiencing issues.

**Comment by DjThossi** [comment]
Hey @juanjoseruiz 

Our first tests showed that this issue is resolved now.
We'll do additional tests in the next weeks (slow due to vacation season).

I'll get back here if there are any other issues popping up.

Until than. Thank you and feel free to have a look at my other feature request.
